Remove expo next references (#2)
powersync-ja/react-native-database-benchmarks: README.md

### Expo-sqlite

> Expo sqlite does not support batching of queries out the box; however expo provides prepared statements to execute a single query multiple times. This is what was used to achieve results similar to batching.

Reviewer: ```suggestion
Expo sqlite does not support batching of queries out the box. Prepared statements are to achieve results similar to batching.
```
